### ISSUE TITLE
[BE] IA mapping: Generate a list of old questions to fields in the database

### ISSUE THREAD

**Issue by Samara-Strauss** [newjersey/dhs-oneapp#148]
## Background

Epic #143

Melissa has undertaken the huge task of mapping the existing IA of OneApp, the new structure we are proposing, and how these two versions map to each other. As part of this, Ron needs to check the stored procedures for validation requirements for quick submit, normal submit, and, if applicable, other submission types (eg. caseworker).

## Tasks

Generate a list of the old question requirements:

- [x] Requirements for quick submit
- [x] Requirements for normal submit
- [x] If applicable, requirements for other submission types
- [ ] Provide this list to Melissa

**Comment by Samara-Strauss** [comment]
@ronaldheft-usds added this ticket to this week since this captures mapping the "required" fields like we talked about today. Here are some additional notes I had from our discussion:

- We need to figure out what the backend considers "required"
  - What are the stored procedures each page is calling?
  - We need to figure out downstream system requirements, but this will be answered once the team can talk to Tom.

**Comment by ronaldheft-usds** [comment]
Documented here, pending verification from Carmen: https://github.com/newjersey/dhs-oneapp/blob/main/docs/validations-and-required-fields.md

TLDR; The current version of OneApp does not enforce field validation in the database, so any and all fields can be marked as required or optional in the new version of OneApp. Downstream requirements TBD and will be discussed as part of #134.

**Comment by Samara-Strauss** [comment]
Will close pending validation from Carmen

**Comment by ronaldheft-usds** [comment]
Carmen validated, and she is beginning to help document the front-end validations: https://github.com/newjersey/dhs-oneapp/blob/main/docs/DFD_Validations/validations-and-required-fields-fromDFD.md
